.github/workflows/setupmc.yml

```diff
@@ -0,0 +1,27 @@
+            - name: Setup Minecraft
+  # You may pin to the exact commit or the version.
+  # uses: mcenv/setup-minecraft@2d092242881552b8ee463d9b4687bd38ef58a396
+  uses: mcenv/setup-minecraft@v3.3.2
+  with:
+    # Minecraft version to use
+    version: 1.12.2
+    # Whether to install Minecraft
+    install: # optional, default is true
+    # Whether to cache Minecraft
+    cache: # optional
+    # Number of retries to download Minecraft
+    retries: # optional, default is 3
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_clarification):** Retries option is not explicitly set.

Although 'retries' has a default value, explicitly setting it could make the workflow's configuration more transparent.

```suggestion
    retries: 3 # optional, default is 3
```